wandb/cli/cli.py

```diff
@@ -601,6 +601,18 @@ def sync_beta(
     "job_type",
     help="Specifies the type of run for grouping related runs together.",
 )
+@click.option(
+    "--add_tag",
+    "add_tag",
+    multiple=True,
+    help="Used to add labels to runs",
+)
+@click.option(
+    "--remove_tag",
+    "remove_tag",
+    multiple=True,
+    help="Used to remove a label from a run",
+)
```

Comment:
The audience for these docs are CLI users. `wandb sync` only syncs one run (afaik), so "to runs" should be "to the run". And "Used to ..." should be changed to active voice, so "Adds a label" or "Removes a label".

* add_tag: "Adds a label to the run."
* remove_tag: "Removes a label from the run."

Also, it looks like "tag" and "label" refer to the same concept. What word do our docs use / what word will users be the most familiar with?

Reply:
@timoffex thanks! will update the changelog. and tags would be the term that users are most familiar with, so will include that